### ISSUE TITLE
Why do I get nREPL 0.2.6 with a fresh install of leinigen?

### ISSUE THREAD

**Issue by maacl** [technomancy/leiningen#1840]
I have upgraded to CIDER 0.9.0 and get incompatibility warning. Tried reinstalling leinigen but still get nREPL 0.2.6.

**Comment by hyPiRion** [comment]
This is because the latest release of Leiningen ships with `0.2.6`.

However, you can (temporarily) override this by putting this in your project.clj:

```clj
:profiles {:dev {:dependencies [[org.clojure/tools.nrepl "0.2.7"
                                 :exclusions [org.clojure/clojure]]]}
```

**Comment by arrdem** [comment]
Is there a good reason not to jump to tools.nrepl 0.2.10 and tag as closed for 2.5.2?

**Comment by hyPiRion** [comment]
@arrdem: Chas went on a release streak – I think I bumped from 2.7 to 2.8 to 2.9 within 3-4 days or so, and then 1-2 days after, Chas released 0.2.10. So I decided to take a small break from bumping until it stopped. (I've bumped it again now).

I've found it helpful to keep stuff open to avoid duplicates when there's an issue that affects many users, but if it's more sensible to close it, I can probably do that. It's solved on master, so it doesn't matter either way.
